templates/applicantForm.phtml

```diff
@@ -57,6 +57,9 @@
                     <select id="cohorts" class="submitApplicant form-control" name="cohortId"  data-selected="<?php echo $applicant->getCohortId();?>" required>
                     </select>
 
+                    <label for="backgroundInfoId">Background Info</label>
+                    <select id="backgroundInfo" class="submitApplicant form-control" name="backgroundInfoId" required></select>
```

Comment:
This field does not show the current value - other dropdown fields have a default option which is whatever is in the db for that applicant, can you set this up to do the same please.

To reproduce just edit an applicant, change their background to school leaver and save, then refresh. The background field will just show Changing careers